tests/integration/test_charm.py

```diff
@@ -56,6 +69,55 @@ async def test_build_and_deploy(ops_test: OpsTest):
     assert ops_test.model.applications[APP_NAME].units[0].workload_status == "active"
 
 
+@pytest.mark.abort_on_fail
+async def test_configmap_created(lightkube_client: Client, ops_test: OpsTest):
+    """Test configmaps contents with default coonfig."""
+    seldon_config_cm = lightkube_client.get(
+        ConfigMap, SELDON_CM_NAME, namespace=ops_test.model_name
+    )
+
+    assert seldon_config_cm.data == SELDON_CONFIG
+
+
+@pytest.mark.abort_on_fail
+async def test_configmap_changes_with_config(lightkube_client: Client, ops_test: OpsTest):
+    await ops_test.model.applications[APP_NAME].set_config(
+        {
+            "custom_images": '{"configmap__predictor__sklearn__v2": "custom:1.0", "configmap_explainer": "custom:2.0"}'  # noqa: E501
+        }
+    )
+    await ops_test.model.wait_for_idle(
+        apps=[APP_NAME], status="active", raise_on_blocked=True, timeout=300
+    )
+    seldon_config_cm = lightkube_client.get(
+        ConfigMap, SELDON_CM_NAME, namespace=ops_test.model_name
+    )
+
+    assert seldon_config_cm.data == SELDON_CONFIG_CHANGED
+
+
+@pytest.mark.abort_on_fail
+async def test_blocked_on_invalid_config(ops_test: OpsTest):
+    await ops_test.model.applications[APP_NAME].set_config({"custom_images": "{"})
+    await ops_test.model.wait_for_idle(
+        apps=[APP_NAME], status="blocked", raise_on_blocked=False, timeout=300
+    )
+    assert ops_test.model.applications[APP_NAME].units[0].workload_status == "blocked"
+
+
+@pytest.mark.abort_on_fail
+async def test_back_to_active_on_config(lightkube_client: Client, ops_test: OpsTest):
```

Comment:
This test case is alright, but I'd change the name to ` test_back_to_active_on_default_config`, as we want a working seldon-core-operator and its activators to be active and functional before we proceed with a bunch of other test cases.